src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -58,9 +59,11 @@ private static (List<Error> errors, List<TableOfContentsInputItem>) LoadTocModel
             else if (filePath.EndsWith(".json", StringComparison.OrdinalIgnoreCase))
             {
                 JToken tocToken;
+                List<Error> nullErrors;
                 try
                 {
-                    tocToken = JToken.Parse(content);
+                    (nullErrors, tocToken) = JsonUtility.Deserialize<JToken>(content);
+                    errors.AddRange(nullErrors);
                 }
                 catch (Exception ex)
```

Comment:
remove the catch here because errors are reported using return values

Reply:
cannot just remove, since catch will take care of json syntax errors, will hide them in JsonUtility instead then

---
In reply to: [200285659](https://github.com/dotnet/docfx/pull/2994#discussion_r200285659) [](ancestors = 200285659)